doc/sphinx/tutorial/write-your-first-wayland-compositor.md

```diff
@@ -2,47 +2,50 @@
 discourse: 4911,5164,5603,6756,8037
 ---
 
-# Write Your First Wayland Compositor
-This tutorial will guide you through writing a simple compositor: the
-installation of dependencies, building it, and running it. You'll be impressed
-by how easy it is to create a Wayland compositor using Mir!
+# Write your first Wayland compositor
+This tutorial will guide you through writing a basic Mir compositor. By the end of it, you will create, build, and run a program with basic window management
+capabilities such as controlling multiple windows, minimizing and maximizing, and handling mouse input. 
 
-## Assumptions
-This tutorial assumes that:
-1. The reader is familiar with C++ and CMake.
-2. The reader has cmake and a C++ compiler installed
-
-## A barebones Mir compositor
 This section will cover the needed dependencies and how to install them, the
 minimum code needed for a Mir compositor, how to build this code, and finally
 how to run your compositor.
 
-### Dependencies
-Before you start coding, you'll need to install `libmiral`, and
-`mir-graphics-drivers-desktop` which can be done on different distros as
-follows:
+## Assumptions
+
+This tutorial assumes that:
+
+* You are familiar with C++ and CMake.
+* You have `cmake` and a C++ compiler installed.
+
+## Installing dependencies
+
+The example program requires 
+* `libmiral`- a library for Mir abstraction layer
```

Comment:
Some suggestions: (looks like I misclicked something when starting the review :))